eoscompanion/dummy_feed.py

```diff
@@ -0,0 +1,166 @@
+# /eoscompanion/dummy_feed.py
+#
+# Copyright (C) 2018 Endless Mobile, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+# All rights reserved.
+'''Function to provide a dummy feed.'''
+
+def dummy_feed():
+    '''Construct a dummy feed.'''
+    return {
+            'status': 'ok',
+            'payload': [
+                {
+                    'sources': [
+                        {
+                            'type': 'application',
+                            'detail': 'com.endlessm.celebrities.en',
+                            'icon': '/v1/application_icon?deviceUUID=1234&iconName=com.endlessm.celebrities.en',
+                            'displayName': 'Celebrities',
+                            'shortDescription': 'Learn more about your favorite celebrities'
+                        },
+                        {
+                            'type': 'application',
+                            'detail': 'com.endlessm.wiki_art.en',
+                            'icon': '/v1/application_icon?deviceUUID=1234&iconName=com.endlessm.wiki_art.en',
+                            'displayName': 'WikiArt',
+                            'shortDescription': 'A collection of great visual works'
+                        },
+                        {
+                            'type': 'application',
+                            'detail': 'com.endlessm.video_kids',
+                            'icon': '/v1/application_icon?iconName=com.endlessm.video_kids&deviceUUID=1234',
+                            'displayName': 'Kids',
+                            'shortDescription': 'Kids will love these videos'
+                        },
+                        {
+                            'type': 'application',
+                            'detail': 'com.endlessm.word_of_the_day.en',
+                            'displayName': 'Word of the day'
```

Comment:
Instead of leaving out `icon` and `shortDescription`- I'd make them blank, since I'm pretty sure the doc says that those keys are guaranteed to be there.

Reply:
Yes, had it first like that - done.